tronbyt_server/manager.py

```diff
@@ -779,19 +779,6 @@ def configapp(device_id: str, iname: str, delete_on_cancel: int) -> ResponseRetu
     abort(HTTPStatus.BAD_REQUEST)
 
 
-@bp.route("/<string:device_id>/brightness", methods=["GET"])
```

Comment:
Why remove this? This is used by TidbytAssistant to expose brightness as a light entity in Home Assistant.

Reply:
didn't realize it was still used. I'll put it back in.